### PR TITLE
Improves README and CONTRIBUTING document to reflect current worflow.

### LINKED ISSUE #103: As a developer I want a better Readme such that my first steps are easier
The Readme should be more conform the industry standard.

For example:
https://bulldogjob.com/news/449-how-to-write-a-good-readme-for-your-github-project
https://www.makeareadme.com/

### PR DESCRIPTION
**Pull request by marickmanrho**
And makes it prettier also. Fixes #103

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+  <a href="https://app.gridt.org">
+    <img src="https://img.shields.io/website?url=https%3A%2F%2Fapp.gridt.org" alt="website">
+  </a>
+  <a href="https://github.com/GridtNetwork/gridt-client/actions?query=workflow%3ACI">
+    <img src="https://github.com/GridtNetwork/gridt-client/workflows/CI/badge.svg?branch=main" alt="Build Status">
+  </a>
+  <a href="https://github.com/GridtNetwork/gridt-client/actions?query=workflow%3ACI">
+    <img src="https://img.shields.io/github/last-commit/GridtNetwork/gridt-client" alt="Build Status">
+  </a>
+</p>
```
Comment (Drvanon) [inline README.md:17]: Why are you going for inline HTML?

Comment (Drvanon) [reply README.md:17]: Correction: SO SHINY!

**Review by Drvanon — APPROVED** [review]
Looking mighty fine!

File: README.md (hunk 1 above)
Comment (marickmanrho) [reply README.md:17]: Centering isn't possible with plain Markdown. That's why.

Comment (Drvanon) [reply README.md:17]: Sensible.

**Pull request merged**